api/app/signals/apps/services/domain/filescanner.py

```diff
@@ -52,6 +53,9 @@ def scan_file(uploaded_file):
         seen.add(UploadScannerService._get_mime_from_content(uploaded_file))
 
         if len(seen) == 1 and list(seen)[0] in ALLOWED_MIME_TYPES:
+            # Let's see if PIL raises the DecompressionBombError
```

Comment:
Move this check to below the more basic checks, in case one of the cheap checks fails first we don't have to load the image.